Run README through Grammarly, rename LICENSE to MIT-LICENSE (#326)
karafka/rdkafka-ruby: README.md

The `rdkafka` gem is a modern Kafka client library for Ruby based on
[librdkafka](https://github.com/edenhill/librdkafka/).
It wraps the production-ready C client using the [ffi](https://github.com/ffi/ffi)
gem and targets Kafka 1.0+ and Ruby versions under security or

> active maintenance. We remove the Ruby version from our CI builds if they 

Reviewer: ```suggestion
active maintenance. We remove a Ruby version from our CI builds when they 
```
